src/coreclr/binder/clrprivbinderassemblyloadcontext.cpp

```diff
@@ -42,19 +42,19 @@ HRESULT CLRPrivBinderAssemblyLoadContext::BindAssemblyByNameWorker(BINDER_SPACE:
     return hr;
 }
 
-HRESULT CLRPrivBinderAssemblyLoadContext::BindAssemblyByName(IAssemblyName     *pIAssemblyName,
+HRESULT CLRPrivBinderAssemblyLoadContext::BindAssemblyByName(struct AssemblyNameData *pAssemblyNameData,
```

Comment:
Yeah, it is a normal pointer because it still goes through the (unfortunately pervasive) `ICLRPrivBinder` interface. Aside from the `ICLRPrivBinder` implementations, `AssemblyName::Init` and `BaseAssemblySpec::PopulateAssemblyNameData` (which do have it by ref) should be the only other places that take it in.

Reply:
Thanks - makes sense.